<?php defined('BASEPATH') OR exit('No direct script access allowed'); ?>

ERROR - 2019-04-26 12:22:54 --> Severity: Warning --> mysqli::real_connect(): (HY000/1049): Unknown database 'oracle' C:\xampp\htdocs\shopify\monika\system\database\drivers\mysqli\mysqli_driver.php 161
ERROR - 2019-04-26 12:22:55 --> Unable to connect to the database
ERROR - 2019-04-26 13:53:44 --> 404 Page Not Found: Cfirst/isAvailable%20
ERROR - 2019-04-26 13:59:30 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 32
ERROR - 2019-04-26 13:59:30 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 13:59:39 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 32
ERROR - 2019-04-26 13:59:39 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 13:59:56 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 32
ERROR - 2019-04-26 13:59:56 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:00:00 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 32
ERROR - 2019-04-26 14:00:00 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:00:28 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 32
ERROR - 2019-04-26 14:00:28 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:01:02 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 32
ERROR - 2019-04-26 14:01:02 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:04:06 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 34
ERROR - 2019-04-26 14:04:06 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:04:17 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 35
ERROR - 2019-04-26 14:04:17 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:07:05 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 35
ERROR - 2019-04-26 14:07:05 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:07:40 --> Severity: error --> Exception: syntax error, unexpected ')' C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 35
ERROR - 2019-04-26 14:07:40 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:34:52 --> 404 Page Not Found: Cfirst/isAvailable%20
ERROR - 2019-04-26 14:41:05 --> Severity: error --> Exception: syntax error, unexpected 'public' (T_PUBLIC), expecting end of file C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 52
ERROR - 2019-04-26 14:41:05 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 14:41:09 --> Severity: error --> Exception: syntax error, unexpected 'public' (T_PUBLIC), expecting end of file C:\xampp\htdocs\shopify\monika\application\controllers\Cfirst.php 52
ERROR - 2019-04-26 14:41:09 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-04-26 15:08:46 --> Severity: Notice --> Undefined property: cfirst::$article C:\xampp\htdocs\shopify\monika\system\core\Model.php 77
ERROR - 2019-04-26 15:08:46 --> Query error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'WHERE `articleid` IS NULL' at line 2 - Invalid query: SELECT `articleid`
WHERE `articleid` IS NULL
